main.py

```diff
@@ -344,9 +425,12 @@ def __init__(self, *args):
 		self.inst = Xqemu()
 		self.settings = SettingsManager()
 		self.settings.load()
-		self.runButton.setText('Start')
-		self.pauseButton.setText('Pause')
-		self.screenshotButton.setText('Screenshot')
```

Comment:
Where did this (or equivalent) move?

Reply:
Its not necessary as far as i can tell since the text will be set after launch due to the `.ui` files anyway